x-pack/plugins/security/server/saved_objects/secure_saved_objects_client_wrapper.ts

```diff
@@ -476,6 +482,39 @@ export class SecureSavedObjectsClientWrapper implements SavedObjectsClientContra
     return await this.redactSavedObjectsNamespaces(response);
   }
 
+  public async removeReferencesTo(
+    type: string,
+    id: string,
+    options: SavedObjectsRemoveReferencesToOptions = {}
+  ) {
+    try {
+      const args = { type, id, options };
+      await this.ensureAuthorized(type, 'delete', options.namespace, {
+        args,
+        auditAction: 'removeReferences',
+      });
+    } catch (error) {
+      this.auditLogger.log(
+        savedObjectEvent({
+          action: SavedObjectAction.REMOVE_REFERENCES,
+          savedObject: { type, id },
+          error,
+        })
+      );
+      throw error;
+    }
+
+    this.auditLogger.log(
+      savedObjectEvent({
+        action: SavedObjectAction.REMOVE_REFERENCES,
+        savedObject: { type, id },
+        outcome: EventOutcome.UNKNOWN,
+      })
+    );
```

Comment:
Brilliant! Thanks for adding audit events for this :)
Looks good from my side!

Reply:
Actually, would be great if you could you add the new event type to the documentation:
https://github.com/elastic/kibana/blob/2b2f85cd223f05e0dbb0305fb2cf3f5b0362200a/docs/user/security/audit-logging.asciidoc#type-change

Reply:
Done in https://github.com/elastic/kibana/pull/79096/commits/7cddbfd081ac6ee52fe0b89be3307d27c8220630